route.go

```diff
@@ -64,7 +64,7 @@ func (r *Route) Match(req *http.Request, match *RouteMatch) bool {
 				match.MatchErr = nil
 			}
 
-			matchErr = nil
+			matchErr = nil // nolint:ineffassign
```

Comment:
Can we just delete this line entirely?

Reply:
I think that we should leave it and we can refactor it later once we understand better what they were trying to do right here.